stage1_fly/run/main.go

```diff
@@ -110,15 +110,24 @@ func init() {
 	flag.StringVar(&discardString, "local-config", common.DefaultLocalConfigDir, "Local config path")
 }
 
+func checkAndAddMps(namedVolumeMounts map[types.ACName]volumeMountTuple, mountpoints []types.MountPoint) error {
```

Comment:
> Maybe even call namedVolumeMounts simply target, and mountpoints simply src.

Would you rename this only in the function or also in the caller? I kept the naming from there.